tests/performance/test_data_analysis/conftest.py

```diff
@@ -0,0 +1,120 @@
+import pytest
```

Comment:
Same here, try to respect pep8, docstrings, etc

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5622/commits/0dcb61333d61863972d8521e0fb586ce9c779a3b